src/code/reducers/drakes.js

```diff
@@ -24,16 +24,15 @@ export default function drakes(state = initialState, action) {
           secondXAlleles = GeneticsUtils.computeExtraAlleles(oldOrg, newOrg);
           alleleString = newOrg.getAlleleString();
         }
-        else if (drakeDef.sex === BioLogica.MALE && action.newSex === BioLogica.FEMALE) {
+        else if (drakeDef.sex === BioLogica.MALE && action.newSex === BioLogica.FEMALE && drakeDef.secondXAlleles) {
```

Comment:
With this change I don't think the `alleleString` gets set when `drakeDef.secondXAlleles` is empty. The test for `drakeDef.secondXAlleles` should be moved inside the braces. You might consider taking a TDD approach: write a unit test for this case before changing the code.